openssl/openssl_funcs.h

```diff
@@ -49,27 +49,22 @@ DEFINEFUNC(int, RAND_bytes, (uint8_t * arg0, size_t arg1), (arg0, arg1)) \
 DEFINEFUNC(int, SHA1_Init, (SHA_CTX * arg0), (arg0)) \
 DEFINEFUNC(int, SHA1_Update, (SHA_CTX * arg0, const void *arg1, size_t arg2), (arg0, arg1, arg2)) \
 DEFINEFUNC(int, SHA1_Final, (uint8_t * arg0, SHA_CTX *arg1), (arg0, arg1)) \
```

Comment:
These three functions are deprecated now in OpenSSL 3. Will these be removed in later commits (reviewing commit by commit as suggested)

Reply:
Yes! In https://github.com/microsoft/go-crypto-openssl/pull/6/commits/aca75befff3521d350bd92494abe5e07748ce940.